README.md

```diff
@@ -1,17 +1,19 @@
 # cdh5-pseudo-docker
 Pseudo-distributed HBase Docker image based on CDH5 including YARN
 
-Uses Java 8 and CDH 5.11.1. Includes HDFS, HBase, and YARN.
+Uses Java 8 (openJDK) and CDH 5.11.1 or 5.16.1. Includes HDFS, HBase, and YARN.
```

Comment:
Surprised this is changing since Cloudera still claims they only want an Oracle JDK for 5.11:

https://docs.cloudera.com/documentation/enterprise/5-11-x/topics/cdh_ig_jdk_installation.html#topic_29

Looks like only 5.16 supports OpenJDK:

https://docs.cloudera.com/documentation/enterprise/5-16-x/topics/cdh_ig_jdk_installation.html#topic_29

Reply:
Yeah - the issue is Oracle change'd it's licensing and the JDK needs paid processor licensing to run in production (and requires accepting of the new policy at install now).   I was able to get 5.11.1 working ok with openJDK at a basic level - I'm not sure exactly what parts don't work.

Do you think it should only setup for >=5.16.1 ?